hasher-matcher-actioner/hmalib/lambdas/api/action_rules_api.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import bottle
+import typing as t
+import sys
+
+from botocore.exceptions import ClientError
+from bottle import response
+from dataclasses import dataclass, field
+from hmalib.common.logging import get_logger
+from .middleware import jsoninator, JSONifiable, DictParseable
+from hmalib.common.config import HMAConfig
+from hmalib.common import config as hmaconfig
+from hmalib.common.evaluator_models import ActionRule
+from hmalib.common.classification_models import Label, ActionLabel
+
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class ActionRulesAPIRequest(DictParseable):
```

Comment:
Naming nit, I saw this in dataset_api as well. Personally, I feel the fact that this file is in the `/api/` folder and the classes have names with `Request` and `Response` makes the use of API in both the base filename and classnames a little superfluous. Thoughts?

Reply:
Sounds good. Nix'd API in the names.